ui/packages/catalog-ui-search/src/main/webapp/react-component/container/table-export/table-export.tsx

```diff
@@ -69,7 +69,15 @@ function getHits(sources: Source[]): number {
     .reduce((hits, source) => (source.hits ? hits + source.hits : hits), 0)
 }
 
-function getExportCount(exportSize: string, selectionInterface: any): number {
+function getExportCount(
+  exportSize: string,
```

Comment:
It would be good to define these in a typescript interface, then destructure an object that uses that interface for the parameter of `getExportCount` and `getWarning`

Reply:
Added the interface as suggested